juju/juju
# Ensure remote offer is exposed when relating cross model, and UI cleanups

When relating to applications across models, both applications potentially need to route to each other. 
As a first step, we ensure both applications are exposed. But this opens both apps to the world, so as a followup we'll need to do some security group changes to tighten things up. Usually, one app is exposed anyway.

As a drive by, remove unnecessary info from status.
Also limit application urls from the CLI to be "model.app", not arbitrary urls as we won't be supporting those.

QA
juju bootstrap aws mine
juju deploy mysql
juju switch controller
juju consume default.mysql
juju deploy mediawiki
juju relate mediawiki:db mysql

Ensure mediawiki page comes up as expected.
